src/component/page/gjennomforing-detaljer/FilterMeny.tsx

```diff
@@ -29,10 +30,14 @@ export const FilterMeny = (props: Props): React.ReactElement => {
 						onChange={(e) => {
 							if (e.target.checked) {
 								leggTilTiltakStatus(status)
+								loggKlikk('FilterMeny', status, 'checked')
```

Comment:
Tenker det hadde vært greit å samle metrikk-taggene 1 sted, f.eks i amplitude-utils.
Litt subjektivt, men for tagger så føler jeg det er bedre med snake case en pascal case.

Reply:
Det ikke er noe poeng i å flytte det ut så lenge man ikke skal bruke den samme taggen flere steder. Kan sikkert bruke snake case